x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/index.tsx

```diff
@@ -327,6 +328,17 @@ const StepAboutRuleComponent: FC<StepAboutRuleProps> = ({
               />
             </EuiFormRow>
             <EuiSpacer size="l" />
+            <EuiFormRow fullWidth>
+              <UseField
+                path="maxSignals"
+                component={MaxSignals}
+                componentProps={{
```

Comment:
`MaxSignals` component defines a `placeholder` prop, but we don't pass it here. I think it'd be a good idea to pass a default value here:
```
placeholder: DEFAULT_MAX_SIGNALS,
```
<img width="217" alt="Scherm­afbeelding 2024-04-29 om 12 23 40" src="https://github.com/elastic/kibana/assets/15949146/840e6c03-e63d-4adb-99bf-8b35e185a181">

Reply:
Yeah that's probably a good idea to address the case you mention above too, I'll switch to that case